css/style.css

```diff
@@ -878,6 +878,20 @@ h2 a {
   border-color: #333;
 }
 
+.content-404 {
+  height: 70vh;
+  padding: 153px 32px 7%;
+  text-align: center;
+  display: flex;
+  align-items: center;

```

Comment:
Maybe just make sure this rule does anything. Seems to center without it. 
` align-items: center;`
Optional remove at your discretion.

Reply:
good catch